Mesh_3/examples/Mesh_3/mesh_3D_image_with_features.cpp

```diff
@@ -79,21 +49,28 @@ int main(int argc, char* argv[])
     return EXIT_FAILURE;
   }
 
-  // Domain
-  Mesh_domain domain = Mesh_domain::create_labeled_image_mesh_domain(image);
-
-  /// Declare 1D-features, see above [Call add_1D_features]
-  const std::string lines_fname = (argc>2)?argv[2]:CGAL::data_file_path("images/420.polylines.txt");
-
-  if(!add_1D_features(image, domain, lines_fname)) {
+  /// Load 1D-features
+  const std::string lines_fname = (argc > 2) ? argv[2] : CGAL::data_file_path("images/420.polylines.txt");
+  std::vector<std::vector<K::Point_3> > features_inside;
+  if (!read_polylines(lines_fname, features_inside)) // see file "read_polylines.h"
+  {
+    std::cerr << "Error: Cannot read file " << lines_fname << std::endl;
     return EXIT_FAILURE;
   }
-  /// [Call add_1D_features]
+
+  /// [Domain creation]
+  Mesh_domain domain = Mesh_domain::create_labeled_image_mesh_domain(image,
+    params::features_detector = CGAL::Mesh_3::Detect_features_on_image_bbox(),
+    params::input_features  = std::cref(features_inside));
```

Comment:
I think there should be a sentence somewhere that expains that use of `std::cref`. That is important.